content/en/docs/contribute/start.md

```diff
@@ -210,15 +210,11 @@ to base your work on. Use these guidelines to make the decision:
     periodically merge these to `master`. This kind of branch has a name like
     dev-\<version>-\<language code>.\<team milestone>; for example:
     `dev-{{< release-branch >}}-ja.1`.
-- If you're writing or updating documentation for a feature change release,
-  then you need to know the major and minor version of Kubernetes that
-  the change will first appear in.
-  - For example, if the feature gate JustAnExample is going to move from alpha
-    to beta in the next minor version, you need to know what the next minor
-    version number is.
-  - Find the release branch named for that version. For example, features that
-    changed in the v{{< release-branch >}} release got documented in the branch
-    named `dev-{{< release-branch >}}`.
+- If you're writing or updating documentation for a future release,
+  then you'll need to know the upcoming Kubernetes release version.
+  - Find the development branch named for that version. For example, features that
```

Comment:
@jimangel OK, though you don't explicitly say how or where to find the `upcoming Kubernetes release version`.
Could also indicate that docs uses this pattern (dev-<release version>) or naming scheme for next release content development branches.
Instead of `Features that changed in the ...`. 
Possible edit:
To document new features or update existing features (as part of a release update)